.circleci/config.yml

```diff
@@ -65,3 +65,94 @@ jobs:
                       cd progress
                       yarn test:demos
                       cd ..
+            - persist_to_workspace:
+                  root: .
+                  paths:
+                    - iconfont
+                    - design
```

Comment:
I don't think we need to persist 'design' folder unless we are doing the build step again later

Reply:
If the design folder isn't something that is generated you're correct, I'll remove it from the persist list.